content/en/docs/ambient/usage/troubleshoot-waypoint/index.md

```diff
@@ -26,14 +26,14 @@ $ kubectl exec deploy/sleep -- curl -s http://$REVIEWS_V2_POD_IP:9080/reviews/1
 Requests to the `reviews` service should be enforced by the `reviews-svc-waypoint` for any L7 policies.
 Requests to the `reviews` `v2` pod should be enforced by the `reviews-v2-pod-waypoint` for any L7 policies.
 
-1.  If your L7 policy isn't enforced, run `istioctl analyze` first to check if your policy has any validation issue.
+1.  If your L7 configuration isn't applied, run `istioctl analyze` first to check if your configuration has a validation issue.
```

Comment:
**nit** "isn't applied" sounds to me like it was rejected by validation in the Kubernetes API but that may just be me so feel free to ignore...
```suggestion
1.  If your L7 configuration isn't working as expected, run `istioctl analyze` first to check if your configuration has a validation issue.
```